docs/tutorial.md

```diff
@@ -61,13 +59,14 @@ helm-summarize --suite v1
 This reads the pre-existing files in `benchmark_output/runs/v1/` that were written by `helm-run` previously, and writes the following new files back to `benchmark_output/runs/v1/`:
 
 - `summary.json` contains a serialized `ExecutiveSummary` with a date and suite name.
+- `run_specs.json` contains the run spec descriptions for all the runs.
 - `runs.json` contains serialized list of `Run`, which contains the run path, run spec and adapter spec and statistics for each run.
 - `groups.json` contains a serialized list of `Table`, each containing information about groups in a group category.
 - `groups_metadata.json` contains a list of all the groups along with a human-readable description and a taxonomy.
 
 Additionally, for each group and group-relavent metric, it will output a pair of files: `benchmark_output/runs/v1/groups/latex/<group_name>_<metric_name>.tex` and `benchmark_output/runs/v1/groups/latex/<group_name>_<metric_name>.json`. These files contain the statistics for that metric from each run within the group.
 
-**Note**: Because `helm-summarize` reads the `run_specs.json` that was produced by `helm-run` and `run_specs.json` is overwritten by every invocation of `helm-run`, `helm-summarize` will only process runs from the most recent invocation of `helm-run`.
+**Note**: Because `helm-summarize` reads the `run_specs.json` files (in each run's subdirectory) produced by `helm-run`, which are overwritten by every invocation of `helm-run`, `helm-summarize` will only process runs from the most recent invocation of `helm-run`.
```

Comment:
Actually we can delete this whole paragraph now. `helm-summarize` now processes all runs in the folder.